### PR TITLE
move to position: fixed

### PR DESCRIPTION
**Pull request by danrschlosser**
Fixes the parallax scrolling so that it doesn't mess up if you reload half way down the page.

Lessons learned:  
- `background-attachement: fixed` is borked in chrome: http://stackoverflow.com/questions/20268962/fixed-attachment-background-image-flicker-disappear-in-chrome-when-coupled-with
- `position: fixed` to the rescue!

### CODE REVIEW

**Comment by natebrennand** [comment]
ping @RaymondXu / @eunicekokor / @evantarrh
(I don't know enough CSS)

File: app/static/js/home/home.js [hunk 1]
```diff
@@ -22,10 +23,19 @@ $(function() {
...
 
+        var translateY = function(y) {
+            return 'translate3d(0px, -' + y + 'px, 0px)'
+        }
+
         var scrollHandler = function() {
             var scrolled = $(window).scrollTop();
             var scrolled = Math.min(scrolled, $image.height());
-            $image.css('transform','translate3d(0px, ' + (scrolled * scroll_factor) + 'px, 0px)');
+            console.log(scrolled, scrolled * scroll_factor);
```
Comment (evantarrh) [inline app/static/js/home/home.js]: remove console log (sorry late nite dan)

File: app/static/js/home/home.js [hunk 2]
```diff
@@ -22,10 +23,19 @@ $(function() {
...
+            $image.css({
+                'transform': translateY(scrolled * scroll_factor),
+                '-moz-transform': translateY(scrolled * scroll_factor),
+                '-webkit-transform': translateY(scrolled * scroll_factor)
+            });
```
Comment (evantarrh) [inline app/static/js/home/home.js]: any reason we're omitting `-o-transform`?

**Comment by danrschlosser** [comment]
@evantarrh updated

**Comment by evantarrh** [comment]
the rest of it looks :+1:

**Pull request merged**